project-base/docker/php-fpm/docker-php-consumer-entrypoint

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+default_tries_before_fail=100
+messages_to_consume_count=1000
+
+consumer_name=$1
+
+if [[ -z ${consumer_name} ]]; then
+    1>&2 echo "First argument had to be name of consumer to start"
+    exit 1
+fi
+
+if [[ -n ${CONSUMER_RUNS_BEFORE_RESTART} ]]; then
+    messages_to_consume_count=${CONSUMER_RUNS_BEFORE_RESTART}
+fi
+
+n=0
+until [[ ${n} -ge ${default_tries_before_fail} ]]; do
+    php bin/console shopsys:rabbitmq:check-availability >/dev/null 2>&1 && php bin/console >/dev/null 2>&1 && break
+    1>&2 echo "Consumer is not yet ready - sleeping"
+    n=$[$n+1]
+    sleep 5
+done
+
+if [[ ${n} = ${default_tries_before_fail} ]]; then
+    1>&2 echo "Number of maximum retries reached - failing"
+    exit 1
+fi
+
+ENVIRONMENT="$(php bin/console about | grep Environment | sed 's/Environment//;s/ //g')"
+
+if [[ ${ENVIRONMENT} = prod ]]; then
+    php bin/console rabbitmq:consumer -m ${messages_to_consume_count} ${consumer_name}
+else
+    until [[ ${n} -ge ${messages_to_consume_count} ]]; do
```

Comment:
Ou, this is a bug. Fixed in f6a9f4721f8287157f825343b73ec2011353ccd9!

Reply:
It's fine now :+1: 

Maybe we can avoid such bugs in the future by using a `for` loop instead of iterating manually:

```bash
for ((i = 0; i < $[tries_before_fail]; i++)); do
    php bin/console shopsys:rabbitmq:check-availability >/dev/null 2>&1 && php bin/console >/dev/null 2>&1 && break
    1>&2 echo "Application is not yet ready - consumer sleeping"
    sleep 5
done
```

```bash
for ((i = 0; i < $[messages_to_consume_count]; i++)); do
    php bin/console rabbitmq:consumer -m 1 ${consumer_name}
done
```

Reply:
Even nicer! Changed in da9568ccebe226f7dced6733ea19ba8453b76563

Reply:
Now you'll need to update the upgrade notes... Sorry for having you jump though all these hoops :man_facepalming:

Reply:
Fixed in ec98a842b362619b52cedd0dbdf7278c6ac832b7 :)

Reply:
Great! Thanks :tada: